src/__tests__/platforms.spec.ts

```diff
@@ -36,6 +39,17 @@ const accountHolderDetails: AccountHolderDetails = {
         country: "NL"
     },
 };
+const notificationConfigurationDetails: NotificationConfigurationDetails = {
+        active: true,
```

Comment:
```suggestion
    active: true,
```

Reply:
Yeah sorry the description is not very clear. It was meant for Ricardo, I just wanted to merge this into a feature branch that both of us are still working on.